.pre-commit-config.yaml

```
@@ -91,7 +91,7 @@ repos:
           - --config=.bandit.yml
   # Run bandit everything but tests directory
```

Can we clean up this comment as long as we're here?
```suggestion
  # Run bandit on everything except the tests directory
```